pandas/core/arrays/string_arrow.py

```diff
@@ -85,7 +85,8 @@ class ArrowStringArray(ArrowExtensionArray, BaseStringArray, ObjectStringArrayMi
 
     See Also
     --------
-    array
+    `array
+    <https://pandas.pydata.org/docs/dev/reference/api/pandas.arrays.ArrowStringArray.html>`__
```

Comment:
Thanks @ramvikrams! I think the intent here is for this to point to `pandas.array` instead of `ArrowStringArray`. Something like the suggestion below should handle that. It also will make it so the link always points to the version of the documentation page that's currently being used. As is, the link in this PR will point to the `dev` version of the docs. 

```suggestion
    pandas.array
```

Reply:
Yes i'll do that